angular/theme.scss

```diff
@@ -47,25 +48,52 @@ $material-theme: mat-light-theme($material-theme-primary, $material-theme-accent
 @include angular-material-theme($material-theme);
 
 // Blue Theme
+@function pxb-light-theme($primary, $accent, $warn) {
+    @return map-merge(
+        mat-light-theme($primary, $accent, $warn),
+        (
+            foreground: (
+                text: map-get($pxb-gray, 800),
```

Comment:
our default text color is supposed to be black[500]

Reply:
Updated the text color

Reply:
Getting some clarification on what our default text color should be.